mitodl/mitxonline
# Migrations to release phase

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/4935

### Description (What does it do?)
This PR switches bin/post_compile migration script to the release phase script.

### How can this be tested?
It will be tested in heroku deployment on RC after merging.
For local testing:

Go to the web bash shell
Run bash scripts/heroku-release-phase.sh
Verify that the script containing migration commands runs successfully.
